src/Datadog.Trace/DogStatsD/TracerMetricNames.cs

```diff
@@ -0,0 +1,73 @@
+namespace Datadog.Trace.DogStatsD
+{
+    internal static class TracerMetricNames
+    {
+        public static class Api
+        {
+            /// <summary>
+            /// Count: Total number of API requests made
+            /// </summary>
+            public const string Requests = "datadog.tracer.api.requests";
+
+            /// <summary>
+            /// Count: Count of API responses.
+            /// This metric has an additional tag of "status: {code}" to group the responses by the HTTP response code.
+            /// This is different from <seealso cref="Errors"/> in that this is all HTTP responses
+            /// regardless of status code, and <seealso cref="Errors"/> is exceptions raised from making an API call.
+            /// </summary>
```

Comment:
Great doc! 👍

Reply:
Thanks, but I just copy/pasted this from @brettlangdon's document! 😄